src/main/resources/de/tracetronic/jenkins/plugins/ecutest/report/generator/ReportGeneratorPublisher/help.html

```diff
@@ -34,7 +37,11 @@
             </pre>
             <pre>
 def instance = ET.newInstallation('ECU-TEST', 'C:\\Program Files\\ECU-TEST 8.0')
-instance.publishGenerators([[name: 'JSON']], [name: 'Custom-JSON')])
+instance.publishGenerators([[name: 'JSON']], [[name: 'Custom-JSON']])
+            </pre>
+            <pre>
+def instance = ET.newInstallation toolName: 'ECU-TEST', installPath: C:\\Program Files\\ECU-TEST 8.0'
+instance.publishGenerators generators: [[name: 'JSON']], customGenerators: [[name: 'Custom-JSON']]
```

Comment:
Other parameters like: `allowMissing`, `runOnFailed`, `archiving` and `keepAll` are not documented in this example.

Reply:
I mean these are just examples, full method syntax is provided above...

Reply:
Listed all archiving options in at least one example.